frontend/components/simulation/SimulationForm.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <form class="flex flex-col md:flex-row gap-4 items-end" @submit.prevent="submitForm">
+        <div class="flex flex-col md:flex-row gap-4 w-full">
+            <label class="w-full md:w-1/2">
+                <TextBlock class="font-semibold text-xs">ChaosLabs access token</TextBlock>
+                <Input v-model="inputToken" :disabled="isLoading || disabled || isStarted" />
+            </label>
+            <label class="w-full md:w-1/2">
+                <TextBlock class="font-semibold text-xs">Simulation id</TextBlock>
+                <Select
+                    :value="selectedSimulationId"
+                    :disabled="isLoading || disabled || noSimulationsAvailable || isStarted"
+                    :loading="isLoading"
+                    :options="simulations"
+                    @change="selectedSimulationId = $event"
+                />
+            </label>
+        </div>
+        <BaseButton
+            :type="hasProperEnvVariables ? 'primary' : ''"
+            class="w-48"
+            :is-loading="isLoading"
+            :disabled="disabled || !hasProperEnvVariables"
+            html-type="submit"
+        >
+            <span v-if="isLoading && isStarted">Resetting</span>
+            <span v-else-if="isLoading">Starting</span>
+            <span v-else-if="isStarted">Reset</span>
+            <span v-else>Start Simulation</span>
+        </BaseButton>
+    </form>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Input, Select } from 'ant-design-vue';
+import BaseButton from '~/components/common/BaseButton.vue';
+import TextBlock from '~/components/common/TextBlock.vue';
+
+export default Vue.extend({
+    components: {
+        Input,
+        BaseButton,
+        Select,
+        TextBlock,
+    },
+    props: {
+        token: {
+            type: String,
+            default: '',
+        },
+        simulationIds: {
+            type: String,
+            default: '',
+        },
+        isStarted: {
+            type: Boolean,
+            default: false,
+        },
+        isLoading: {
+            type: Boolean,
+            default: false,
+        },
+        disabled: {
+            type: Boolean,
+            default: false,
+        },
+    },
+    data() {
+        return {
+            inputToken: this.token,
+            selectedSimulationId: this.simulationIds.split(',')[0],
+        };
+    },
+    computed: {
+        simulations(): Object {
```

Comment:
As far as I can see, `simulations` returns an array of [`SelectOption`s](https://github.com/sidestream-tech/unified-auctions-ui/blob/eed9564bc4b4d64afbe241f45cbcb690b55639ba/frontend/shims.d.ts#L25-L29). Please avoid using `Object` contractor as type for this exact reasons: in js arrays are also objects, as well as functions, etc